source/val/validation_state.cpp

```diff
@@ -829,4 +845,20 @@ const std::vector<uint32_t>& ValidationState_t::FunctionEntryPoints(
   }
 }
 
+std::string ValidationState_t::Disassemble(const Instruction& inst) const {
+  const spv_parsed_instruction_t& c_inst(inst.c_inst());
+  return Disassemble(c_inst.words, c_inst.num_words);
+}
+
+std::string ValidationState_t::Disassemble(const uint32_t* words,
+                                           uint16_t num_words) const {
+  uint32_t disassembly_options = SPV_BINARY_TO_TEXT_OPTION_NO_HEADER |
+                                 SPV_BINARY_TO_TEXT_OPTION_FRIENDLY_NAMES |
+                                 SPV_BINARY_TO_TEXT_OPTION_SHOW_BYTE_OFFSET;
```

Comment:
Actually, if the strategy changes to have the whole binary in hand, and you really do know which instruction position (index) it is, then you should keep this show-byte-offset because then it's the truth and extra information the user could use.

Reply:
Removed for now. It was always outputting 0x14 and I couldn't figure out where it came from. We can always put back later if desired.